drivers/SmartThings/zwave-switch/profiles/child-generic-sensor.yml

```diff
@@ -0,0 +1,40 @@
+name: child-generic-sensor
+components:
+- id: main
+  capabilities:
+  # Notification CC based capabilities
+  - id: contactSensor
+    version: 1
+  - id: motionSensor
+    version: 1
+  - id: smokeDetector
+    version: 1
+  - id: waterSensor
+    version: 1
+  - id: tamperAlert
+    version: 1
+  # Sensor Multilevel CC based capabilities
+  - id: atmosphericPressureMeasurement
+    version: 1
+  - id: bodyWeightMeasurement
+    version: 1
+  - id: illuminanceMeasurement
+    version: 1
+  - id: relativeHumidityMeasurement
+    version: 1
+  - id: temperatureMeasurement
+    version: 1
+  - id: voltageMeasurement
+    version: 1
+  # Meter CC based capabilities
+  - id: energyMeter
+    version: 1
+  - id: powerMeter
+    version: 1
```

Comment:
for the time being I'd stick to the capabilities handled by the DTH implementation, namely: smoke, motion, water, and contact

Reply:
Ok, there are any additional capabilities? I don't know how the default DTH implementation handles Sensor reports.